VideoLocker/src/main/java/org/edx/mobile/view/CourseDashboardFragment.java

```diff
@@ -30,111 +28,124 @@
     static public String CourseData = TAG + ".course_data";
 
     private EnrolledCoursesResponse courseData;
+    private boolean IS_COURSEWARE_ACCESSIBLE = true;
```

Comment:
This all caps naming convention is usually reserved for constants, so I suggest changing this variable to the more standard camel case format.